src/elements/Modal/Modal.js

```diff
@@ -17,7 +17,8 @@ class Modal extends Component {
 		dequeueModal: PropTypes.func.isRequired,
 		modalRoot: PropTypes.instanceOf(window.Element).isRequired,
 		className: PropTypes.string,
-		dismissible: PropTypes.bool
+		dismissible: PropTypes.bool,
+		full: PropTypes.bool
```

Comment:
if we're going to have this be a toggle, we should name it appropriately `fullScreenMobile` or similar to disambiguate from a full screen modal at any screen size

Reply:
So it's not actually just for mobile, you can make it fullscreen for all of it (or, if you wanted it to be specifically for one or the other, you could use it conjunction with `isMobile()`). But maybe `fullScreen` is still a better name for it?

Reply:
A full screen modal on desktop isn't a modal, it's an overlay. I would have this switch only for mobile, if needed.

I would agree with an argument to never have modals be full screen, and if we need an overlay just for mobile we would create and use an `Overlay` ui component

Reply:
Alright, I've made the `fullScreen` prop only work for mobile